packages/@expo/cli/src/start/startAsync.ts

```diff
@@ -72,12 +73,16 @@ export async function startAsync(
   if (exp.platforms?.includes('ios') && process.platform !== 'win32') {
     // If Xcode could potentially be used, then we should eagerly perform the
     // assertions since they can take a while on cold boots.
-    getXcodeVersionAsync({ silent: true });
+    await getXcodeVersionAsync({ silent: true });
```

Comment:
Not awaiting this was intentional, see the logs above.

```suggestion
    getXcodeVersionAsync({ silent: true });
```